### PR TITLE
Adding ExecutionDefinition generation POC

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

File: ExecDef/tools.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import re
+from pathlib import Path
+
+from gooddata_pandas import GoodPandas
+from gooddata_sdk import Attribute, ExecutionDefinition, GoodDataSdk, ObjId, SimpleMetric
+
+
+def get_workspace_id() -> str:
+    sdk = GoodDataSdk.create_from_profile(profile="default", profiles_path=Path("./profiles.yaml"))
+    return sdk.catalog_workspace.list_workspaces()[0].workspace_id
```
Comment (hkad98) [inline ExecDef/tools.py:11]: Not sure about this. Why do you pick the first workspace from the list of workspaces?

File: ExecDef/tools.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    This is a bulldozer of a function.
+
+    Args:
+        answer (str): Answer from the OpenAI agent.
+
+    Returns:
+        dict: _description_
+    """
+    regex = r"\{(?:[^{}]|)*\}"
```
Comment (hkad98) [inline ExecDef/tools.py:25]: Consider using re.compile to improve performance. It is a tip for a future. No need to do that now ;)

File: ExecDef/tools.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    Args:
+        answer (str):
+            Answer from the OpenAI agent, can be either a valid .json or
+            a written text containing the valid .json
+    """
+
+    exdef = answer_to_json(answer)
+
+    gp = GoodPandas("http://localhost:3000", "YWRtaW46Ym9vdHN0cmFwOmFkbWluMTIz")
```
Comment (hkad98) [inline ExecDef/tools.py:46]: Use create from profile class method. It should already be working for the used version.

File: ExecDef/tools.py (hunk 1 above)
Comment (Mara3l) [reply ExecDef/tools.py:11]: Just for simplicity, as this presumes, that the user will only have demo as their only workspace.

File: ExecDef/tools.py (hunk 2 above)
Comment (Mara3l) [reply ExecDef/tools.py:25]: Thanks! :)

File: ExecDef/tools.py (hunk 3 above)
Comment (Mara3l) [reply ExecDef/tools.py:46]: Could you please provide the snippet?

**Pull request merged**